docs/src/Services/Reference/PhpDocHelper.php

```diff
@@ -53,22 +53,23 @@ public function handleClassDoc(\ReflectionClass $class, string $content): string
     {
         $rawDocNode = $class->getDocComment();
 
-        if ($rawDocNode) {
-            $tokens = new TokenIterator($this->lexer->tokenize($rawDocNode));
-            $phpDocNode = $this->parser->parse($tokens);
-            $tokens->consumeTokenType(Lexer::TOKEN_END);
-            $text = array_filter($phpDocNode->children, static function (PhpDocChildNode $child): bool {
-                return $child instanceof PhpDocTextNode;
-            });
-
-            /** @var PhpDocTextNode $t */
-            foreach ($text as $t) {
-                // todo {@see ... } breaks generation, but we can probably reference it better
-                if (str_contains($t->text, '@see')) {
-                    $t = str_replace(['{@see', '}'], ['see', ''], $t->text);
-                }
-                $content .= $t.\PHP_EOL;
+        if (!$rawDocNode) {
+            return $content;
+        }
+        $tokens = new TokenIterator($this->lexer->tokenize($rawDocNode));
+        $phpDocNode = $this->parser->parse($tokens);
+        $tokens->consumeTokenType(Lexer::TOKEN_END);
+        $text = array_filter($phpDocNode->children, static function (PhpDocChildNode $child): bool {
+            return $child instanceof PhpDocTextNode;
+        });
+
+        /** @var PhpDocTextNode $t */
+        foreach ($text as $t) {
+            // todo {@see ... } breaks generation, but we can probably reference it better
```

Comment:
this is only because of `mdx` and is valid in `md` so let's just don't handle this here we should handle this in the next.js config :)

Reply:
Noted !
I think we could also parse and link it (php side) if it references existing pages if you want me to handle that .